glue_solar/__init__.py

```diff
@@ -1,12 +1,15 @@
 from pkg_resources import get_distribution, DistributionNotFound
+from sunpy.cm import cmlist
 
 try:
     __version__ = get_distribution(__name__).version
 except DistributionNotFound:
     pass
 
-
 def setup():
     from glue.viewers.image.qt import ImageViewer
```

Comment:
@mbobra can you move all these imports out of the setup function and to the top of the file.